Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -169,7 +169,7 @@
   icon: { sprite: /Textures/Objects/Misc/bureaucracy.rsi, state: pen }
   productEntity: PenExplodingBox
   cost:
-    Telecrystal: 5
+    Telecrystal: 4
```

Comment:
pen prob shouldnt cost more than c4

Reply:
pen was more expensive than c4 both before and after c4 nerf so i dunno, just left it like that because it has some novelty as a stealth item, i'll get to these eventually either